github-user-card-react/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import UserCard from './Components/userCard';
+import UserFollowers from './Components/userFollowers';
+
+import './App.css';
+
+class App extends React.Component {
+  constructor(){
+    super()
+    this.state = {
+      users : []
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://api.github.com/users/irisjitomo')
+    .then(res => res.json())
+    .then(res => this.setState({ users : res}))
+    .catch(err => console.log("err"));
+  }
```

Comment:
Instead of logging the string "err" when there is an error, go ahead and send the the `err` obj to the console.

`.catch(err => console.log(err)`